docs/appendix.rst

```diff
@@ -524,8 +509,145 @@ To fix this, install the Python "wheel" package::
+# for socket objects that implemented PEP 644,  however, we never had any issues
+# with new minimum TLS version until 3.1.12 so the path through requests was
+# not impacted or we would have had to add urllib 2.0 in earlier tests.  It appears
+# that for the paths we use, it is requests, urllib3, pyOpenSSL packages and
+# OpenSSL version that are the issue.  I need to do another code review.
+Note that Python starting version 3.10 requires OpenSSL version 1.1.1 or higher
+(see Python PEP-644).
+
+Because of this change issue (Warnings, Exceptions, etc.) can occur in the
+process of:
+
+* pywbem package installation
+* pywbem package update ( ie.e updating urllib3)
+* OS update which changes the SSL implementaton version
+* Pywbem attempting to communicate with a WBEM server where the server may
+  not implement the new TLS version requirements.
+
+The following documents present more information on this change to urllib3 and
+the issues and possible corrections:
+
+* `Python PEP 644 – Require OpenSSL 1.1.1 or newer <https://peps.python.org/pep-0644/>`_
+  which defines the future direction of SSL in the Python world.
+
+* `Urllib3 migration guide <https://urllib3.readthedocs.io/en/stable/v2-migration-guide.html>`_
+
+* `macOS issues with Urllib3 - urllib3 issue 3024 <https://github.com/urllib3/urllib3/issues/3020>`_
+
+* `urllib3 issue 2168, Drop support for OpenSS\<1.1.1 <https://github.com/urllib3/urllib3/issues/2168>`_
+
+The current version and implementation  of the SSL library can be determined
+as follows::
+
+  $ python -c "import ssl; print(ssl.OPENSSL_VERSION)"
+  OpenSSL 1.1.1  7 Feb 2023
+
+  or
+
+  $ /usr/bin/python3 -c "import ssl; print(ssl.OPENSSL_VERSION)"
+  LibreSSL 2.8.3
+
+OpenSSL version can be directly determined from OpenSSL as follows::
+
+    $ openssl version
+    OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
+
+The current urllib3 installed version can be determined with pip::
+
+  $ pip list | grep urllib3
+  urllib3                       2.2.1
+
+The following subsections define specific issues that can occur with this
+change and proposed solutions.
+
+* :ref:`NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+`
+* :ref:`ConnectionError raised with [SSL] EC lib`
+* :ref:`ConnectionError raised with [SSL: UNSUPPORTED_PROTOCOL]`
+* :ref:`ImportError urllib3 v2.0 only supports OpenSSL 1.1.1+`
+
+.. _`NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+`:
+
+NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. index:: pair; installation fail: NotOpenSSLWarning
+.. index:: pair; troubleshooting: NotOpenSSLWarning
+
+This issue may be caused by the urllib3 package update to version 2.0
+in pywbem version 1.7.0. In this case it is probably that the local environment
+includes a version of OpenSSL less than 2.0 The best options is to
```

Comment:
"a version of OpenSSL less than 2.0"
Should that not be "a version of OpenSSL less than 1.1.1"?

Reply:
DONE